addons/website_sale/views/templates.xml

```diff
@@ -1382,10 +1382,10 @@
                                         <strong t-field="line.name_short" />
                                     </div>
                                 </td>
-                                <td class='td-qty'>
+                                <td class='text-right td-qty'>
                                     <div t-esc="line.product_uom_qty" />
                                 </td>
-                                <td class="text-center td-price">
+                                <td class="text-right td-price pr-2">
```

Comment:
Why adding `pr-2` here ?

Reply:
there is one point in spec to align product price list and subtotal in the right column 
and its generic for all themes and also for the community 
for reference: https://drive.google.com/file/d/14LAe-uTMnZ2ZdlCltPhet0C2hYt1lDRf/view

Reply:
Ok but
- It is not aligned with `pr-2`
- Even if it was, it is not the correct way to do so. The problem comes from the two tables (one for the product, one for the total) using different sizing (one table-sm, the other not). The solution is either of those:
    - Do not use `table-sm` for both
    - Use `table-sm` for both
I prefer the first solution but maybe @JKE-be has a different opinion? I know this box is tricky...

Reply:
I have done change as per your first solution :)